icu4c/source/data/unit/de.txt

```diff
@@ -10,12 +10,42 @@ de{
     units{
         acceleration{
             g-force{
```

Comment:
Using unit/de.txt as an exemplar for what I'm doing with the ldml2icu rules:

In this PR's current state, with its suggested ldml2icu rules, I'm adding "case" and "gender" right next to the plural forms.

This means they must be handled by the PluralTableSink: I'm skipping "case" at this time. When case is specified, its forms should be used instead of the normal (nominative) plural forms. I'm keeping "gender", as with "dnam" and "per".